build/windows/omsupply-prepare.bat

```diff
@@ -0,0 +1,3 @@
+@ECHO ##### Prepare omsupply build #####
+REM cd "..\..\client" && yarn install --force && yarn build && cd "..\server" && cargo clean
+cd "..\..\client" && yarn install --force && yarn build && cd "..\server" && cargo clean

```

Comment:
i don't think we need to  do `cargo clean` here, and also `yarn install --force --frozen-lockfile`

Reply:
thanks Andrei - popped that in too 😉